tracee-rules/benchmark/benchmark_test.go

```diff
@@ -0,0 +1,143 @@
+package benchmark
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aquasecurity/tracee/tracee-rules/benchmark/signature/golang"
+	"github.com/aquasecurity/tracee/tracee-rules/benchmark/signature/rego"
+	"github.com/aquasecurity/tracee/tracee-rules/engine"
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	inputEventsCount = 1000
+)
+
+func BenchmarkOnEventCodeInjectionRegoRule(b *testing.B) {
+	codeInjectSig, err := rego.NewCodeInjectionSignature()
+	require.NoError(b, err)
+
+	err = codeInjectSig.Init(ignoreFinding)
+	require.NoError(b, err)
+
```

Comment:
We need a `b.ResetTimer()` call here. 


<img width="640" alt="image" src="https://user-images.githubusercontent.com/1254783/123193230-3ae20180-d459-11eb-9be4-492f8fd8c5fd.png">


Same for all other benchmarks.